beacon-chain/sync/subscriber_beacon_attestation.go

```diff
@@ -46,34 +46,30 @@ func (r *Service) committeeIndexBeaconAttestationSubscriber(ctx context.Context,
 	return r.attPool.SaveUnaggregatedAttestation(a)
 }
 
-func (r *Service) committeesCount() int {
-	activeValidatorIndices, err := r.chain.HeadValidatorsIndices(helpers.SlotToEpoch(r.chain.HeadSlot()))
-	if err != nil {
-		panic(err)
-	}
-	return int(helpers.SlotCommitteeCount(uint64(len(activeValidatorIndices))))
+func (r *Service) subnetCount() int {
```

Comment:
Probably can just remove this method if it only reads the config value.

Reply:
Taking up